livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamHomeActivity.kt

```diff
@@ -75,32 +69,46 @@ class LiveStreamHomeActivity : ComponentActivity() {
                                     .height(24.dp)
                             )
                         }
-                        //Theme chooser drop down selector
+                        // Theme chooser drop down selector
                         DropdownMenu(
                             expanded = expanded,
                             onDismissRequest = { expanded = false }
                         ) {
-
-                            DropdownMenuItem(onClick = {
-                                expanded = !expanded
-                                isDarkMode = false
-                            }) {
-                                Text(text = "LiveStream light theme")
+                            DropdownMenuItem(
+                                onClick = {
+                                    expanded = !expanded
+                                    isDarkMode = false
+                                    updateTheme(isDarkMode)
+                                }
+                            ) {
+                                Text(text = stringResource(R.string.light_theme_toggle_title))
                             }
-                            DropdownMenuItem(onClick = {
-                                expanded = !expanded
-                                isDarkMode = true
-                            }) {
-                                Text(text = "LiveStream dark theme")
+                            DropdownMenuItem(
+                                onClick = {
+                                    expanded = !expanded
+                                    isDarkMode = true
+                                    updateTheme(isDarkMode)
+                                }
+                            ) {
+                                Text(text = stringResource(R.string.dark_theme_toggle_title))
                             }
                         }
 
-                        //Adds a icon button for switching grid toggle
-                        IconButton(onClick = {
-                            isGrid = !isGrid
-                        }) {
+                        // Adds a icon button for switching grid toggle
+                        IconButton(
+                            onClick = {
+                                isGrid = !isGrid
+                            }
+                        ) {
+                            // We also update the icon when the state changes from a grid view to single list
+                            val iconToShow =
+                                if (!isGrid) {
```

Comment:
why not `painterResource(if(!isGrid) oneValue else otherValue)`

Reply:
Updated

Reply:
Closing this comment , in lieu of updated code 

this comment is pointing to outdated code